tasks/dd_apm_logs_config.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Create Datadog logging directories
+  file:
+    path: "/etc/datadog-agent/conf.d/{{ item }}.d/"
+    state: directory
+    mode: 0755
+    owner: dd-agent
+    group: dd-agent
+  loop: "{{ sonar_log_names }}"
+  tags:
+    - datadog
+
+- name: Create Datadog log config files.
+  template:
+    src: datadog-logs-config.yml
+    dest: "/etc/datadog-agent/conf.d/{{ item }}.d/config.yaml"
+    mode: 0755
+    owner: dd-agent
+    group: dd-agent
+  loop: "{{ sonar_log_names }}"
+  notify: restart datadog-agent
+  tags:
+    - datadog
+
+- name: Download Datadog APM agent
+  get_url:
+    url: https://repo.dfw.rtrdc.net/rtr/global/dd-java-agent-0.33.0.jar
+    dest: /etc/datadog-agent/dd-java-agent.jar
```

Comment:
Consider storing the java APM agent in `/opt/datadog-java-agent` like we do in other repos: https://github.com/RentTheRunway/deploy-tools/blob/master/playbooks/datadog-java-apm.yml and https://github.com/RentTheRunway/jenkins/pull/145/files#diff-85ac0832e76659694a50ea483e581a81